images/installer/Dockerfile

```diff
@@ -26,15 +26,16 @@ COPY images/installer/root /
 RUN yum install -y epel-release 'dnf-command(config-manager)' && \
     yum config-manager --enable built > /dev/null && \
     yum install --setopt=tsflags=nodocs -y \
-      'ansible-core < 2.14' \
       python38-dateutil python38-urllib3 \
```

Comment:
```suggestion
      python39 python39-devel python39-urllib3 \
```

Reply:
I believe that python needs to be updated in order to grab a higher version of `ansible-core`.